res/css/views/right_panel/_TimelineCard.scss

```diff
@@ -33,4 +33,55 @@ limitations under the License.
             right: 0;
```

Comment:
From [@toger5's demo](https://matrix.to/#/!kxwQeJPhRigXSZrHqf:matrix.org/$Mwb4wKGLgGdimhg6-DcpIkZCLHgOWWQXjP7HvBf3pTU?via=matrix.org&via=element.io&via=vector.modular.im) of maximized widgets where it showed the right-panel chat style, I was surprised that it looked a lot more native and refined compared to the thread view. This probably comes down to the differences from the normal timeline view because we're changing some the pieces around like the timestamp position in the thread view. I understand that the right panel chat style and the thread view are different.

Right panel chat style that feels nice to me:
![](https://user-images.githubusercontent.com/558581/145890527-5a51b9ff-f1dd-4a48-b631-f34a79efcc05.png)

The unpolished feeling may apply to this PR iteration of the right-panel chat where things are getting condensed like the thread view but it's hard to judge because I haven't actually used it in the app like threads and it's nice to be able to judge the balance against the whole app and timeline view.

As @toger5 notes, some adaptations probably need to be made for the smaller layout,

> We definitely have to make changes to the style. Just to get read-receipts not glitching/overlapping with the messages. The question is, if the style changes are too aggressive. I see both sides. It looking more like the main timeline (more native) when changing as a little as possible and adapting it to the right panel so the small area is used as efficient as possible.
>
> *-- @toger5, https://matrix.to/#/!tCaHWuHHvOHjWyqapP:matrix.org/$UibGBNcTpdbe-0zdRn1Ktp8_yHWYh9wL6HlfxJ3t-tc?via=matrix.org&via=element.io&via=uhoreg.ca*


---

As a tiny point of reference, Slack and Discord are able to maintain the same layout between the main message view and threads but they have a different timestamp position and less UI (don't need to contend with read-receipts).